projects/core/directives/dropdown/dropdown.component.ts

```diff
@@ -90,7 +90,14 @@ export class TuiDropdownComponent {
     private getStyles(top: number, left: number): Record<string, string> {
         const {right} = this.el.getBoundingClientRect();
         const {maxHeight, minHeight, offset, limitWidth} = this.options;
-        const {innerHeight} = this.win;
+        const innerHeight =
+            parseInt(
+                this.win
```

Comment:
This variable just comes from `win.visualViewport` object, there's no need to get CSS var value with a heavy `getComputedStyle` operation

Reply:
fixed